docker-compose.override.prod.yml

```diff
@@ -1,18 +1,22 @@
 version: "3"
 
 services:
-  # In production, dockerize the frontend
+  # In production, use registry image
+  elasticsearch:
+    image: registry.gitlab.factory.social.gouv.fr/socialgouv/code-du-travail-numerique/elasticsearch:$CDTN_HASH
+    ports:
```

Comment:
9200

Reply:
Also set a défaut when CDTN_HASH not provided ?

Reply:
there is no master tag so we should provide an arbitrary one ?

Reply:
latest ?

Reply:
nor latest ;)

Reply:
i guess latest should exist by convention ?